bin/conjur-cli.rb

```diff
@@ -18,40 +19,39 @@ def test_select
       false
     end
   end
-  
+
   30.times do
     break if test_select
     $stderr.write '.'
     sleep 1
   end
 
   raise "Database is still unavailable. Aborting!" unless test_select
-  
+
   true
 end
 
 desc 'Run the application server'
 command :server do |c|
   c.desc 'Account to initialize'
-  c.arg_name 'account'
+  c.arg :account
```

Comment:
This actually doesn't do anything at all for flags -- it's for command arguments -- you should remove them. You can put `arg_name` back in where it makes sense (it doesn't when you just repeat the flag; I'd do `arg_name :name` for account, `arg_name :path` for file, maybe `arg_name :ip` for address. Port should be pretty self-explanatory. (Do watch how the help text changes as you set these.)